test/test_utils.py

```diff
@@ -38,4 +40,8 @@ def parse_common_options(datadir=None,
   xla_folder = os.path.dirname(os.path.dirname(os.path.abspath(sys.argv[0])))
   sys.path.append(os.path.join(os.path.dirname(xla_folder), 'test'))
   sys.path.insert(0, xla_folder)
+  if log_args:
```

Comment:
Why do we need this?
Some of the models won't even use the whole argument set, so it seems a bit weird.

Reply:
For logging purposes, I had found it useful so I can go back read logs and check what the hyperparams were for a certain run, especially for the periodic tests we have run with Cloud TPUs. Right now from the logs I wouldn't be able to tell what the hyperparams used actually were. Just a utility so not strictly needed though.

Reply:
I'd prefer a more explicit `print()` in the model source code, using only the arguments which make sense for that model, instead of dumping everything.